### PR TITLE
AMP-60: PLP heading

### PR DESCRIPTION
**Pull request by cole-sanderson**
Work for PLP heading, this PR will not complete the whole AMP-60 task, still need to add some work on "filter" and "sort by".
 
 **JIRA**: https://mobify.atlassian.net/browse/AMP-60

## Changes
- add `<Breadcrumbs>` component 
- `<ProductList>` container 
- copy and paste category images from `/web` into `/amp`
- delete `mobify.png` image

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions PLP](http://localhost:3000/potions.html)
- Compare PLP Header exclude filter/sort by from AMP/PWA.

### CODE REVIEW

File: amp/app/containers/product-list/container.scss [hunk 1]
```diff
@@ -29,11 +36,15 @@
...
+
+
 // AMP SDK Custom Styles
 // ---
 
-@import '../../styles/themes/amp-components/amp-image';
-@import '../../styles/themes/amp-components/price';
+@import '../../styles/themes/amp-components/breadcrumbs'; // TODO: dont move to SDK
+
+@import '../../styles/themes/amp-components/nav-menu'; // TODO: dont move to SDK
```
Comment (yourpalsonja) [inline amp/app/containers/product-list/container.scss]: This is already getting imported in `app/_base.scss`, we can remove it here.

File: amp/app/containers/product-list/container.jsx [hunk 2]
```diff
@@ -45,7 +47,11 @@ const ProductList = ({
 
 ProductList.propTypes = {
     products: PropTypes.array.isRequired,
-    numItems: PropTypes.number
+    numItems: PropTypes.number,
+    /**
+     * A title
+     */
+    title: PropTypes.string
```
Comment (kschmidtdev) [inline amp/app/containers/product-list/container.jsx]: I don't think this is used

File: amp/app/containers/product-list/partials/product-list-header.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                src={staticURL(`/img/categories/${title.trim().replace(/\s+/g, '-')
+                .toLowerCase()}@2x.png`)}
+                layout="fixed"
+            />
+        }
+    </div>
+)
+
+ProductListHeader.propTypes = {
+    contentsLoaded: PropTypes.bool,
```
Comment (kschmidtdev) [inline amp/app/containers/product-list/partials/product-list-header.jsx]: I don't think this is used

File: amp/app/containers/product-list/partials/product-list-header.jsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    parentHref: PropTypes.string,
+    parentName: PropTypes.string,
+    title: PropTypes.string
+}
+
+const mapStateToProps = createPropsSelector({
+    contentsLoaded: getProductListContentsLoaded,
```
Comment (kschmidtdev) [inline amp/app/containers/product-list/partials/product-list-header.jsx]: Similarly not used

File: amp/app/containers/product-list/partials/product-list-header.jsx [hunk 5]
```diff
@@ -0,0 +1,50 @@
+import React, {PropTypes} from 'react'
+import {connect} from 'react-redux'
+import {createPropsSelector} from 'reselect-immutable-helpers'
+import {getCategoryTitle, getCategoryParentTitle, getCategoryParentHref} from '../../../../../web/app/store/categories/selectors'
+import {getProductListContentsLoaded} from '../../../../../web/app/containers/product-list/selectors'
```
Comment (kschmidtdev) [inline amp/app/containers/product-list/partials/product-list-header.jsx]: We don't need to bring in the contents loaded selector

Comment (cole-sanderson) [inline amp/app/containers/product-list/partials/product-list-header.jsx]: oops, good catch :)

**Pull request merged**